stable_baselines3/dqn/policies.py

```diff
@@ -0,0 +1,267 @@
+from typing import Optional, List, Callable, Union, Type, Any, Dict
+
+import gym
+import numpy as np
+import torch as th
+import torch.nn as nn
+from stable_baselines3.common.policies import (BasePolicy, register_policy, create_mlp,
+                                               NatureCNN, BaseFeaturesExtractor, FlattenExtractor)
+
+
+class QNetwork(BasePolicy):
```

Comment:
Couldn't this processing be done in the Policy code? The "network" part of Q-learning is just the feature_extractor plus linear mapping to outputs size, so I do not think we need separate class for this, since it is not required.

Reply:
true, this was mostly do avoid code duplication when creating the target. It also mimics what is done in SAC/TD3.

Reply:
After thinking more about this code, the main issue is the feature extraction part.
If we remove that class, it makes it harder to properly have a target network...